### PR TITLE
Update fr.po

### PR DESCRIPTION
**Pull request by Vavas7**
Réduction de certaines lignes, ajout des traductions manquantes.
Reduced some strings, added translation on missing ones.

### CODE REVIEW

File: fr.po [hunk 1]
```diff
@@ -4379,11 +4379,11 @@ msgstr ""
 
 #. heading
 msgid "Pick Topic"
-msgstr "Choix du thème"
+msgstr "Thème"
```
Comment (patrickklug) [inline fr.po]: was this a problem in the UI? the previous string doesn't seem too long and might be clearer to the player?

File: fr.po [hunk 2]
```diff
@@ -4379,11 +4379,11 @@ msgstr ""
...
 
 #. heading
 msgid "Pick Platform"
-msgstr "Choix de plateforme"
+msgstr "Plateforme"
```
Comment (patrickklug) [inline fr.po]: was this a problem in the UI? the previous string doesn't seem too long and might be clearer to the player?

**Comment by patrickklug** [comment]
@Vavas7 looks good. just two comments re. the 'Pick Theme/Platform' changes.

File: fr.po (hunk 2 above)
Comment (Vavas7) [inline fr.po]: In fact is make too much. Usually the games are translated this way. So i tried to make it look more professionnal. The player will indeed know that he has to choose the thing.

**Comment by Vavas7** [comment]
That will look better this way, i'm sure of it. If you don't want it, then i can change it. But i'm pretty much sure that people will like it better this way.

**Comment by patrickklug** [comment]
ok, thanks.

**Pull request merged**